assets/scss/settings/fonts.scss

```diff
@@ -0,0 +1,11 @@
+$Georgia : Georgia;
+$Palatino : Palatino;
+$Times_New_Roman : Times;
+$Arial : Arial;
```

Comment:
Name of variable `Arial` should be written in all lowercase letters with hyphens instead of underscores<br>Variable names should be followed immediately by a colon